R/utils_accessibility.R

```diff
@@ -118,9 +118,12 @@ h6_tabstop <- function(header, tabindex = 0, ...) {
 #'
 #' @examples
 #' nhs_card_tabstop("A card", shiny::p("Some content"))
-nhs_card_tabstop <- function(header, tabindex = 0, ...) {
+nhs_card_tabstop <- function(heading, tabindex = 0, ...) {
   # create nhs_card as typical nhs_card plus tabindex attribute
   # ensures nhs_card will be stopped at when pressing keyboard tab
-  nhs_card_tabstop <- div(nhs_card(header, ...)) %>%
-    htmltools::tagAppendAttributes(`tabindex` = tabindex)
+  htmltools::tagQuery(nhs_card(heading, ...))$
+    find(".nhsuk-card__heading")$
+    removeAttr("tabindex")$
+    addAttr("tabindex" = tabindex)$
+    allTags() # Exclude Linting
```

Comment:
My preferred way of mutating HTML. I am not sure why the former code was adding a 0, but this fixes it.